package.json

```diff
@@ -6,8 +6,8 @@
   "ascMain": "assembly/index.ts",
   "types": "assembly/index.ts",
   "scripts": {
-    "pretest": "npm run asbuild:empty",
     "test": "asp --verbose --nologo && npm run test:empty",
+    "pretest:empty": "npm run asbuild:empty",
```

Comment:
Why does this work, you may ask.

`pre` and `post` hooks can be added to any npm script by just prefixing it with `pre` or `post`.
https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts